### PR TITLE
yaaw: Add package

### PR DESCRIPTION
**Pull request by kuoruan**
Yet Another Aria2 Web Frontend in pure HTML/CSS/Javascirpt.

Signed-off-by: Hsing-Wang Liao <kuoruan@gmail.com>

### CODE REVIEW

File: net/yaaw/Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yaaw
+PKG_VERSION:=2016-05-28
+PKG_RELEASE:=1
+PKG_MAINTAINER:=nanpuyue <nanpuyue@gmail.com>
```
Comment (yousong) [inline net/yaaw/Makefile]: Hmm, if you are going to maintain this package, then `PKG_MAINTAINER` needs to be your REAL name.

File: net/yaaw/Makefile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/binux/yaaw.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=c28cde2252b707fe0661693f78c5e5e3beee09a1
+PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.gz
+
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=http://www.gnu.org/licenses/lgpl.txt
```
Comment (yousong) [inline net/yaaw/Makefile]: Quoting the [Contributing Guidelines](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain)

```
An optional PKG_LICENSE_FILES tag including the filenames of the license-files in the source-package. (E.g.: PKG_LICENSE_FILES:=COPYING)
```

**Comment by yousong** [comment]
Though I am not a lawyer and it may not a required option, please consider using real name in the `Signed-off-by` line.

**Comment by kuoruan** [comment]
Is this OK? but there is no license-files in the source-package.

**Comment by hnyman** [comment]
Does this depend on #2756 ?

And I think that Luci https://github.com/openwrt/luci/pull/732 needs this first, right?

**Comment by kuoruan** [comment]
Yaaw and webui-aria2 depends on aria2.

**Comment by yousong** [comment]
I have played with yaaw previously.  If I understand it correctly, it's a standalone web application doing ajax and running only in browser.

**Comment by kuoruan** [comment]
英语不好...

File: net/yaaw/Makefile [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+PKG_NAME:=yaaw
+PKG_VERSION:=2016-05-28
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Hsing-Wang Liao <kuoruan@gmail.com>
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/binux/yaaw.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=c28cde2252b707fe0661693f78c5e5e3beee09a1
+PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.gz
```
Comment (yousong) [inline net/yaaw/Makefile]: Please consider using `$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz` as the name of tarball.  This also applies to the pr about `webui-aria2`.

Comment (kuoruan) [inline net/yaaw/Makefile]: Modified.

**Comment by yousong** [comment]
Thanks

**Pull request merged**